### DISCUSSION TITLE
Limiting recordings disk usage on a Synology docker installation

### DISCUSSION THREAD

**Discussion by iliyang** [koush/scrypted#1468] [Q&A]
I'm testing Scrypted NVR on a Synology docker container. Currently I have it set to write the recordings on a shared folder that I use for other things. But I see the NVR is eating up space pretty quickly, and I want to limit its usage. According to the [documentation](https://docs.scrypted.app/scrypted-nvr/installation.html#recording-storage), I have two options:

1. Have a retention days policy. That's an easy option, but for the life of me I can't figure out where to set the number of days to retain. I don't see it in the settings page of the Scrypted NVR plugin (see 1st screenshot below).

2. Set a hard limit on the partition used by the NVR. I have a only a single volume on my Synology NAS, and I'd like to keep it that way. I can create a shared folder with a limited quota (see 2nd screenshot), but the docs say that "Scrypted NVR will not work with filesystem quota features." Do Synology's limited-space shared folders on fall under this unsupported category?

So at this point I don't know how to make either option work. Any help please?

<img width="827" alt="image" src="https://github.com/koush/scrypted/assets/802594/3b3f8678-084a-4992-a30a-8dc32e7a7ac2">

<img width="877" alt="image" src="https://github.com/koush/scrypted/assets/802594/5d204fe6-5d46-40db-8161-585fdd1b2c5e">

**Comment by koush** [comment]
1. Click save after setting retention days. Then set the days field that shows up. This UI is a bit clunky.
2. For technical reasons, there's no way to limit the storage used. Create a partition instead.
